docker-qgis/entrypoint.py

```diff
@@ -146,9 +147,20 @@ def _extract_layer_data(project_filename: Union[str, Path]) -> dict:
     return layers_by_id
 
 
-def _open_project(project_filename: Union[str, Path]):
+def _open_project(project_filename: str) -> QgsProject:
```

Comment:
I think I'd rename this function to _open_read_only_project to avoid someone else using it in the future not appreciating the read-only nature of this method.

Reply:
Ideally I want to get the arguments already from the Workflow framework, but for now I will rename the function